CHANGELOG.md

```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Added
+- Add subpath exports ([#214](https://github.com/MetaMask/eth-sig-util/pull/214), [#211](https://github.com/MetaMask/eth-sig-util/pull/211))
```

Comment:
I think this is breaking, because the `exports` field prevents you (at least under some circumstances) from requiring or importing modules that aren't listed.

Reply:
Done in https://github.com/MetaMask/eth-sig-util/pull/217/commits/4a08cf7667a2731161e4665612fe2f9f561aea2a